docs/usage-cli.rst

```diff
@@ -68,3 +68,94 @@ If you’re using ``zsh``, add the following to your ``.zshrc``:
   autoload bashcompinit
   bashcompinit
   eval "$(register-python-argcomplete airflow)"
+
+Exporting DAGs to images
+------------------------
+
+The application has the functionality of saving DAG to image file. You can attach them to the documentation
+for the documentation, or send another without having to send the DAG file and install the application at
+the other person. However, you need to have `Graphviz <https://graphviz.gitlab.io/download/>`_ installed.
+
+For example, if you want to export ``example_gcp_datacatalog`` DAG then you can use the following command:
+
+.. code-block:: bash
+
+  airflow dag show example_gcp_datacatalog
+
+After passing the ``dag_id`` parameter itself, the command will print DAG to the screen in the DOT format.
+
+It is possible to save the file in a different format. To do this, add the switch ``--save [filename].[format]``.
+If you want to save files as PNG, you can use the following command:
+
+.. code-block:: bash
+
+  airflow dags show example_gcp_datacatalog --save example_gcp_datacatalog.png
+
+An example image file may look as follow:
+
+.. figure:: img/usage_cli_export.png
+    :width: 100%
+
+    Example DAG representation
+
+The following file formats are supported:
```

Comment:
How about putting that list in single coma separated list ? It will take less space then.

Reply:
It will be very difficult to find the format because each format has a different length. For me, a longer list is not a problem because it can be easily scrolled over.

Reply:
OK.